deploy/operator.yaml

```
@@ -21,8 +21,8 @@ spec:
       containers:
         - name: redhat-marketplace-operator
           # Replace this with the built image name
-          image: redhat-marketplace-operator
-          imagePullPolicy: IfNotPresent
+          image: quay.io/mxpaspa/redhat-marketplace-operator:0.1.4
+          imagePullPolicy: Always
```

Revert to old image:
```suggestion
          image: redhat-marketplace-operator
          imagePullPolicy: IfNotPresent
```